mirascope/chat/types.py

```diff
@@ -112,7 +112,14 @@ def content(self) -> Optional[str]:
 
     @property
     def tool_calls(self) -> Optional[list[ChoiceDeltaToolCall]]:
-        """Returns the tool calls for the 0th choice message."""
+        """Returns the partial tool calls for the 0th choice message.
+
+        The size of `list[ChoiceDeltaToolCall]` most likely never exceeds 1. The first
```

Comment:
Preface this with. `We're making what we think is a reasonable assumption here when we use this call ourselves: ...`